linter-requirements.txt

```diff
@@ -1,10 +1,9 @@
 black==22.3.0
-flake8==3.9.2
-flake8-import-order==0.18.1
```

Comment:
this wasn't actually being executed (since it wasn't part of the `select` list below) so I removed it

Reply:
ok